### PR TITLE
Update email address on support page

### PR DESCRIPTION
**Pull request by jollery**
Jess's email address had an incorrect apostrophe, this is the fix.

### CODE REVIEW

File: source/support.html.erb [hunk 1]
```diff
@@ -126,7 +126,7 @@ title: Support
           tom.dolan@digital.cabinet-office-gov.uk
         </a>
         <br />
-        Jess O’Leary, product manager -
+        Jess O'Leary, product manager -
```
Comment (bleach) [inline source/support.html.erb]: You also need to make the same fix to the following two lines.

Comment (quis) [reply source/support.html.erb]: This is a correct apostrophe and shouldn’t be changed (except in the email address where it won’t work) – see http://smartquotesforsmartpeople.com

Comment (bleach) [reply source/support.html.erb]: @jollery you could alternatively remove the right hand quote marks from your email address entirely. As I just tested, gmail doesn't care if they're there or not and delivers them anyway. It depends if you mind or not.

I'm sure @quis is correct about them being the right quote mark in written language.

Comment (jollery) [reply source/support.html.erb]: Ok, thanks both. I'll remove the apostrophe from the email address entirely.

**Pull request merged**